packages/react/src/DefaultResourceTimeline/DefaultResourceTimeline.tsx

```diff
@@ -12,7 +12,12 @@ export function DefaultResourceTimeline(props: DefaultResourceTimelineProps): JS
     <ResourceTimeline
       value={props.resource}
       loadTimelineResources={async (medplum: MedplumClient, resourceType: ResourceType, id: string) => {
-        return Promise.allSettled([medplum.readHistory(resourceType, id)]);
+        const ref = `${resourceType}/${id}`;
+        const _count = 100;
+        return Promise.allSettled([
+          medplum.readHistory(resourceType, id),
+          medplum.search('Task', { subject: ref, _count }),
```

Comment:
We should also find Tasks whose focus is the current resource

Reply:
https://github.com/medplum/medplum/pull/1961/commits/51df88a773f21cba66e19d220bee61a6bd1b7c86